cpp/examples/abm_minimal.cpp

```diff
@@ -19,70 +19,45 @@
 */
 #include "abm/abm.h"
 #include "abm/household.h"
-#include <cstdio>
 #include <fstream>
 #include <string>
 #include <iostream>
 
-void write_results_to_file(const mio::abm::Simulation& sim)
-{
-    // The results are saved in a table with 9 rows.
-    // The first row is t = time, the others correspond to the number of people with a certain infection state at this time:
-    // S = Susceptible, E = Exposed, I_NS = InfectedNoSymptoms, I_Sy = InfectedSymptoms, I_Sev = InfectedSevere,
-    // I_Crit = InfectedCritical, R = Recovered, D = Dead
-    std::ofstream myfile("abm_minimal.txt", std::ios::out);
-    myfile << "# t S E I_NS I_Sy I_Sev I_Crit R D\n";
-    for (auto i = 0; i < sim.get_result().get_num_time_points(); ++i) {
-        myfile << sim.get_result().get_time(i);
-        auto v = sim.get_result().get_value(i);
-        for (auto j = 0; j < v.size(); ++j) {
-            myfile << v[j];
-            if (j < v.size() - 1) {
-                myfile << " ";
-            }
-        }
-        if (i < sim.get_result().get_num_time_points() - 1) {
-            myfile << "\n";
-        }
-    }
-    std::cout << "Results written to abm_minimal.txt" << std::endl;
-}
-
 int main()
 {
     // This is a minimal example with children and adults < 60 year old.
-    // We divided them into 3 different age groups, which are defined as follows:
-    size_t NUM_AGE_GROUPS         = 4;
-    const auto AGE_GROUP_0_TO_4   = mio::AgeGroup(NUM_AGE_GROUPS - 4);
-    const auto AGE_GROUP_5_TO_14  = mio::AgeGroup(NUM_AGE_GROUPS - 3);
-    const auto AGE_GROUP_15_TO_34 = mio::AgeGroup(NUM_AGE_GROUPS - 2);
-    const auto AGE_GROUP_35_TO_59 = mio::AgeGroup(NUM_AGE_GROUPS - 1);
+    // We divided them into 4 different age groups, which are defined as follows:
+    size_t num_age_groups         = 4;
+    const auto age_group_0_to_4   = mio::AgeGroup(0);
+    const auto age_group_5_to_14  = mio::AgeGroup(1);
+    const auto age_group_15_to_34 = mio::AgeGroup(2);
+    const auto age_group_35_to_59 = mio::AgeGroup(3);
```

Comment:
It's good to adhere to naming conventions. I know that this issue is only about the abm minimal example, but we have the same naming style in the tests and the other examples/simulations. Can you change it there, too, or create a new issue?

Reply:
see #846